src/main/app/src/app/zaken/zaak-view/zaak-view.component.ts

```diff
@@ -356,7 +356,7 @@ export class ZaakViewComponent extends ActionsViewComponent implements OnInit, A
             }, 'upload_file'));
         }
 
-        if (this.zaak.isOpen && !this.zaak.besluit && this.zaak.isBesluittypeAanwezig && this.zaak.rechten.vastleggenBesluit) {
+        if (this.zaak.isOpen && !this.zaak.besluit && !this.isIntake(this.zaak) && this.zaak.isBesluittypeAanwezig && this.zaak.rechten.vastleggenBesluit) {
```

Comment:
Nu wordt aan de voorkant gecontroleerd of de zaak een intake is. Maar gebeurd dat ook aan de backend? 
 
Als er zowel aan de backend als aan de front end gecontroleerd moet worden of de Zaak in de intake status is ... hoe zorg je er dan voor dat de buisness logica om te controleren of de zaak in de intake fase zit op 1 plaats zit? 

Denlk daar eens over na en overeg met mij eventueel meerdere oplossingen

Reply:
Via Teams besproken en geimplementeerd